docs/src/01adr-cli.md

```diff
@@ -0,0 +1,88 @@
+# ADR-01: Atomkraft CLI
+
+| authors           | revision | revision date  |
+| ----------------- | --------:| --------------:|
+| Andrey Kuprianov  |        1 | July 11, 2022  |
+
+This ADR describes the command-line interface of Atomkraft, and how it is supposed to be used by the users as a standalone tool. It *does not* describe other possible usage scenarios, e.g. integration with testing frameworks (such as Pytest). We structure the description around the tool use cases.
```

Comment:
Atomkraft CLI should only be similar to a scaffolding tool for a pytest project. ([ignite-cli](https://github.com/ignite/cli) to [cosmos-sdk](https://github.com/cosmos/cosmos-sdk))

Once the project is setup with Atomkraft, a user can also use pytest to perform tests.

Atomkraft CLI can have some out-of-box commands wrapping pytest.